### PR TITLE
feat: added new SignMessage for signinf simple string message and message to flag signed typed data are not support

### PR DESCRIPTION
**Pull request by GaetanoMondelli**

### CODE REVIEW

**Review by traylewin — COMMENTED** [review]
Please update Chain members as requested in updated Chain interface
Please check all comments are correct
Approved after making these changes

File: src/plugin/algoSignMessage.ts
Comment (traylewin) [inline src/plugin/algoSignMessage.ts:26]: This value should be 'default' so that it is the same across all chains:
e.g. SignMethod.Default - 'default'
Use an Enum and export the type like other types

File: src/plugin/models/cryptoModels.ts
Comment (traylewin) [inline src/plugin/models/cryptoModels.ts:32]: Please move to generalModels

File: src/plugin/models/generalModels.ts
Comment (traylewin) [inline src/plugin/models/generalModels.ts:41]: since SignMethod and AlgoSignDataInput are both only used for signMessage, this should be clear here. please rename:
SignMethod => AlgorandSignMessageMethod
AlgoSignDataInput => AlgorandSignMessageDataInput 
note: please dont shorten the prefix from Algorand - as 'Algo' is the currency name and can add confusion

**Review by traylewin — COMMENTED** [review]
Approved after minor changes - feel free to merge to master

**Review by traylewin — APPROVED** [review]
Approved after minor changes are made - feel free to merge to master after

File: src/plugin/models/generalModels.ts
Comment (GaetanoMondelli) [reply src/plugin/models/generalModels.ts]: Done!
Noted, I got confused with the file naming convention, will keep this in mind.

**Pull request merged**
